src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,31 @@
+$footer-bg-color: #26266d;
+$footer-text-color: white;
+$footer-padding: 20px;
+$logo-width: 150px;
+$logo-margin-right: 20px;
```

Comment:
It'll be better if you create a separate variables.scss to store these
And then import same file in each component scss file